### PR TITLE
fix some whitespace and indentation

### PR DESCRIPTION
**Pull request by jeg139**

### CODE REVIEW

**Comment by michaelforney** [comment]
Maybe a duplicate of #192. Perhaps we could combine our efforts?

File: sshfs.c [hunk 1]
```diff
@@ -568,8 +568,8 @@ static inline void buf_init(struct buffer *buf, size_t size)
 		}
 	} else
 		buf->p = NULL;
-	buf->len = 0;
-	buf->size = size;
+		buf->len = 0;
+		buf->size = size;
```
Comment (michaelforney) [inline sshfs.c]: I think this is wrong, there are no curly braces on the else statement, these two lines run in both cases.

File: sshfs.c [hunk 2]
```diff
@@ -46,9 +46,9 @@
 #include <grp.h>
 #include <limits.h>
 #ifdef __APPLE__
-#  include <strings.h>
-#  include <libgen.h>
-#  include <darwin_compat.h>
+#include <strings.h>
+#include <libgen.h>
+#include <darwin_compat.h>
```
Comment (michaelforney) [inline sshfs.c]: I don't think there is any issue here, the space is used as an indent to show that these files are included only when `__APPLE__` is defined.

**Comment by jeg139** [comment]
Yes we can i was thinking #192 have already be pulled in. No problem.

File: sshfs.c (hunk 1 above)
Comment (jeg139) [reply sshfs.c]: Your alright i'll modify it.

File: sshfs.c (hunk 2 above)
Comment (jeg139) [reply sshfs.c]: Ok i'll modify it.

**Comment by Nikratio** [comment]
I've just merged the older request. Could you please rebase on this one (if further changes are needed), or close this request?

Thanks for submitting this!

**Comment by Nikratio** [comment]
Hmm. Looks like it still applied cleanly, nevermind!

**Pull request merged**